src/js/modules/components/moveable-mixin.js

```diff
@@ -0,0 +1,50 @@
+export const MOVE_END = 'moveend';
+export const MOVE_START = 'movestart';
+const PREVENT_SELECT_CLASS = 'noselect';
+
+function moveableMixin() {
+  let isDragging = false;
+  let element;
+  const offset = {};
+  const position = { x: 0, y: 0 };
```

Comment:
there is no reason to initialize `position` and to not initialize `offset` here.